ops/gulp/docs-js.js

```diff
@@ -0,0 +1,27 @@
+const gulp = require('gulp');
+const gp_concat = require('gulp-concat');
+const gp_rename = require('gulp-rename');
+const gp_uglify = require('gulp-uglify');
+const gulpif = require('gulp-if');
+const gulp_jslint = require('gulp-jslint');
+
+let environment = require('../lib/environment');
+
+const paths = {
+    src: './src/scripts',
+    dest: './www/js'
+}
+
+let techneDestJS = environment.production ? 'techne-all.min.js' : 'techne-all.js';
+
+const task = (cb) => {
+    return gulp.src([`${paths.src}/*`])
+        .pipe(gulp_jslint())
+        .pipe(gp_concat('concat.js'))
```

Comment:
when using gulp-concat files will be concatenated in the order that they are specified in the gulp.src function.If we need some custom includes we can specify it in gulp.src or use https://www.npmjs.com/package/gulp-order. But for now I'd leave it as it is. For now I can't oversee the future exceptions.